addons/website_event_track/static/src/scss/event_track_templates_online.scss

```diff
@@ -50,9 +50,9 @@
         .o_wesession_track_main {
             // Force side panel min-width to account for potential Youtube chat
             // And adapt main panel max-width accordingly.
-            @include media-breakpoint-up(md) {
-                max-width: calc(100% - 22rem);
-            }
+            // @include media-breakpoint-up(md) {
+            //     max-width: calc(100% - 22rem);
+            // }
```

Comment:
:thinking:

Reply:
Added a todo comment, to check after if it's still necessary, as it's breaking the layout

Reply:
For reference, this was introduced for youtube live chat to display properly

@morm-odoo  I tested it and this does seem to make the chat unusable on smaller screens (basically anything smaller than a laptop screen horizontally). Some buttons gradually disappear and we can't really read the chat.

Is there any way we can keep this without breaking your design?
Maybe we can add a class conditionally only when the youtube chat is available?

If difficult to fix I guess it can be merged and this handled with a bugfix but it would be best to handle it now.